Complete the French i18n for the Android mobile application

dilox opened the pull request.

henviale committed:
fix: corrected some typos

henviale committed:
fix: corrected some typos

dcremonini committed:
Amend French i18n

silicia-apps committed:
Add some basic debug functions

silicia-apps committed:
Merge remote-tracking branch 'noiapp/develop' into noiapp-debug

integrazione funzionalità di debug

silicia-apps committed:
add force sync button and autorestart tracing over reset

silicia-apps committed:
fix a lint error

silicia-apps committed:
fix some other errors

dcremonini reviewed (changes requested):
Hi @henviale 
thank you for your contribution.
This pull request is for French language but you committed a file for the English one.
Please may yoy reroute the same content towards an another pull request?

dcremonini committed:
Fix some typos

mauriziopapini committed:
[FIX] Necessario reboot device dopo 'Ripristino SDK'

mauriziopapini committed:
[UPD] extract version from git using describe

see https://github.com/gladed/gradle-android-git-version for template
customization.

NOTE: github CI isn't happy for versionCode extracted so I left it a
fixed constant

silicia-apps committed:
rimossi file non cambiati e aggiunta traduzione francese e tedesco per il tasto

silicia-apps committed:
some other fix

Magicianred committed:
Merge pull request #39 from mauriziopapini/fix/versioning

[UPD] extract version from git using describe

Magicianred committed:
Merge pull request #25 from silicia-apps/noiapp-debug

Add PopUp PING-PONG bluetooth fra device

mauriziopapini committed:
aligning fix/resetSDK

dcremonini committed:
Merge remote-tracking branch 'dp3t-dp3t-app-android/develop' into develop

# Conflicts:
#	app/src/main/res/values-fr/strings.xml
#	app/src/main/res/values-it/strings.xml

Magicianred committed:
Merge pull request #29 from mauriziopapini/fix/resetSDK

[FIX] Necessario reboot device dopo 'Ripristino SDK'

dcremonini committed:
Merge remote-tracking branch 'dcremonini-noi-app-android/develop' into develop

# Conflicts:
#	app/src/main/res/values-fr/strings.xml
#	app/src/main/res/values-it/strings.xml
